macros/schema_tests/column_values_basic/expect_column_values_to_have_consistent_casing.sql

```diff
@@ -0,0 +1,36 @@
+{% test expect_column_values_to_have_consistent_casing(model, column_name, display_inconsistent_columns=False) %}
+
+with test_data as (
+
+     select
+     distinct {{ column_name }} as distinct_values
+     from
+     {{ model }}
+     where 1=1
+
+ ),
+ {% if display_inconsistent_columns %}
```

Comment:
I'm not sure we need to have two paths here? I think the `display_inconsistent_columns` path is sufficient, no? Any benefit to users of the test in the other case?

Reply:
My reasoning was to do with performance. The path that does not display inconsistent columns saves a group by that may be costly when testing huge tables. 

Also, some common tests (like the "column has no nulls") return only one row so I thought doing the same was consistent.

Let me know what you think.

Reply:
Got it, makes sense. Thanks!